src/promptflow/promptflow/_sdk/_service/apis/ui.py

```diff
@@ -16,3 +38,46 @@ def get(self):
             render_template("index.html", url_for=url_for),
             mimetype="text/html",
         )
+
+
+def save_image(base64_data, extension):
+    image_data = base64.b64decode(base64_data)
+    filename = str(uuid.uuid4())
```

Comment:
shall this filename be a hash of its content?

Reply:
updated